R/dbi-driver.R

```diff
@@ -63,8 +63,11 @@ setMethod("show", "OdbcDriver",
 #'   name for the OdbcConnection object returned from [dbConnect()]. However, if
 #'   the driver does not return a valid value, it can be set manually with this
 #'   parameter.
-#' @param attributes An S4 object of connection attributes that are passed
+#' @param attributes A list of connection attributes that are passed
 #'   prior to the connection being established. See \link{ConnectionAttributes}.
+#' @param interruptibleExecution Logical.  If `TRUE` calls to SQLExecute and
```

Comment:
Should we mark this as experimental, and to file an issue on GitHub if it's used? I assume we'll eventually take this away once we're confident that it's ok.